game_board.py

```diff
@@ -148,60 +148,120 @@ def which_color_pair(stone_marker):
         color = 3
     return color
 
-def move_down(plus_list,current_row):
-    """ Finds the "+" char that is below the current_row. If the current "+" char is at the bottom, 
-        it finds the "+" char at the top with the same x position
-    Returns the new y position of the "+" 
+def move_down(plus_list,loc):
+    """ Finds the "+" char that is above loc. If the current "+" char is at a coordinate that
+        that violates the rules for movement, the positiono wont change.
+    Returns the new x position of the "+"  
 
-    Keyword arguments:
-    current_row -- the currently selected row in the plus_list
+
+    loc -- the currently selected position in the plus_list
     plus_list -- the lists of all the "+" and their positions
     """
-    
-    new_row = current_row
+    new_loc = loc
     
     while True:
-        new_row += 1
-        if new_row == len(plus_list):
-            new_row = 0
+        new_loc += 1
```

Comment:
It's better if you write out the full name of the variable. It makes the code more readable
```suggestion
        new_location += 1
```